### PR TITLE
Fix package.json issue

### PR DESCRIPTION
**Pull request by RsanabriaZ**

### CODE REVIEW

**Review by scattrell — APPROVED** [review]
Looks good to me.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "version": "0.1.0",
+  "name": "com.bluefletch.motorola",
+  "cordova_name": "com.bluefletch.motorola",
+  "description": "Motorola Cordova Plugin",
+  "license": "MIT",
+  "repo": "https://github.com/RsanabriaZ/flq-cordova-plugin-laserscanner.git",
```
Comment (ryates8) [inline package.json]: Don't use your public repo here. If this repo has been forked over to the foodlogiq organization, then please update this line to use the flq repo.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  ],
+  "cordova": {
+    "id": "com.bluefletch.motorola",
+    "platforms": [
+      "android"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RsanabriaZ/flq-cordova-plugin-laserscanner.git"
```
Comment (ryates8) [inline package.json]: Don't use your public repo here. If this repo has been forked over to the foodlogiq organization, then please update this line to use the flq repo.

Comment (RsanabriaZ) [reply package.json]: I just copied the package.json content from mine and forgot to change.. I will update it now. Thank you @ryates8

**Comment by dixonmyles** [comment]
You guys may want to move this repo into the private FoodLogiQ directory if it's actively being developed. I haven't worked here for two years but because I did the original fork of this plugin I'm still getting all of your changes and updates. Just a thought :) say hi to the team for me!

**Pull request merged**